app/graphql/types/verse_type.rb

```diff
@@ -0,0 +1,9 @@
+module Types
```

Comment:
I think we also had to create this type, may lead to a conflict

Reply:
We definitely did, although the conflict may just be how the fields are ordered